source/PlayerInfo.h

```diff
@@ -352,6 +354,7 @@ class PlayerInfo {
 	const Planet *planet = nullptr;
 	bool shouldLaunch = false;
 	bool isDead = false;
+	bool ownsCarrier = false;
```

Comment:
This raises a couple of questions:

* How do we keep this counter accurate when a player reloads or quits and restarts the game?  Count on load or reload?
* Should this counter be active escorts or total player inventory (including parked ships)?

We could turn this into a counter but I'm not sure how I would use it.  Some guidance for how to proceed with it being a counter would be good.

We can just update this to a counter later as well when a need is identified.  What are your thoughts on this or my questions?

Reply:
Alternatively, we could change the variable name to be more descriptive, since, as noted, it isn't *exactly* a measure of whether you own carriers. `displayCarrierHelp` seems fitting enough to me.

Reply:
This is a good idea.  I will change it.